plugin-localization.md

```diff
@@ -33,6 +35,19 @@ The **lang.php** file should define and return an array of any depth, for exampl
             'tagline' => 'Getting back to basics'
         ]
     ];
+    
+The **validation.php** file has a similar structure to the **lang.php**  and is used to specify your [custom validation](https://octobercms.com/docs/services/validation#localization) messages in a language file, for example:
+
+    <?php
+
+    return [
+    'custom' => [
```

Comment:
Please fix the formatting on this, and ensure that it reflects the changes made in the other section of the documentation in regards to the validation language file array structure.